doc/source/_includes/rllib/rlm_learner_migration_banner.rst

```diff
@@ -0,0 +1,4 @@
+.. note::
+
+    From Ray 2.6.0 onwards, RLlib is adopting a new stack for training and model customization, moving away from ModelV2 API and some convoluted parts of Policy API.
+    Starting from PPO algorithm, we will be gradually replace components with `RLModule API <rllib-rlmodule.html>`__.
```

Comment:
I'm not familiar with all the technical details, but will this be sufficient for RLlib users to know what's going on?

A few thoughts/questions:
1. The wording of "a new stack" feels internal/ambiguous as a user, maybe you can explicitly call out RLModules and Learners in this sentence instead?
2. "Starting from PPO algorithm"  - what does this mean to the user? Should they expect a new API for PPO in 2.6.0? 
3. If all the information is captures in the RLModule doc, maybe it'll be helpful to just say something like "For more details about this migration(?) see RLModules<...>"